osf/management/commands/force_archive.py

```diff
@@ -129,6 +129,13 @@
 }
 
 def complete_archive_target(reg, addon_short_name):
+    # Update files_count on registration
```

Comment:
@pattisdr restarting failed/stuck registration is not done asynchronously with tasks, hence why `files_count` is updated here as well.

Reply:
OK! Just one quick change, can you please DRY this up - a function you call in both places?